### DISCUSSION TITLE
A `WebException` logged me in as a guest(?)

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#14382] [Q&A] [answered]
Not sure how it happened, but I got this 525 `WebException` (I guess that's the HTTP status code?).
Everything looked ok (e.g. the carousel leaderboard was asking me to log in), except for the user panel, which showed me as a registered user:

![osu_2021-08-18_17-18-48](https://user-images.githubusercontent.com/43073074/129926595-836f0786-33d5-4b11-a966-8b44084cf3e3.png)

I didn't check if there were any other problems. I "logged out" and re-logged in, and everything was back to normal.

Logs:
- [`network.log`](https://github.com/ppy/osu/files/7008377/network.log)
- The [`runtime.log`](https://github.com/ppy/osu/files/7008398/runtime.log) looks clean

Version: `2021.815.0`

**Comment by bdach** [comment, marked as answer]
This needs to be handled as an actual issue, so I've made it one.

**Reply by frenzibyte** [reply]
#14386
